### DISCUSSION TITLE
Browser Fairy abandoned

### DISCUSSION THREAD

**Discussion by ylluminate** [dmarmor/epichrome#292] [General]
From my experience and what I've read so far, just thought folks would want to know that the browser selector that I have personally quite liked and was very reliable while I've been using it, Browser Fairy, appears to have been abandoned by its developers.

I'm currently on the hunt for something better (might build one myself depending on what I find since so far I've not found *all* the functionality I've been looking for).  If you have (or anyone here has) found a particularly compelling browser picker, please let me know.

**Comment by talleux** [comment]
Hello @ylluminate 

have you tried Choosy? I use it since years and i'm happy with it. Works fine on Mac Catalina with Epichrome 2.3 and 2.4 beta.
BTW: Choosy has a much bigger feature set than Browser Fairy, which was my reason for choosing Choosy.

If you need more infos, feel free to ask me.

**Comment by ylluminate** [comment]
@talleux appreciate the pointer - I was aware of it, but for some reasons I felt it was not as functional as Browser Fairy years ago when I started using it. Looking at the release notes history is, though, encouraging: 
https://www.choosyosx.com/releases

Appreciate your bringing that back into my attention and encouraging me to take a closer look!  We'll leave this open a little longer in case anyone else has some input... 😄

**Comment by talleux** [comment]
@ylluminate happy to hear that. The combination of epichrome and choosy is on my top 5, well, top 1 of my daily productivity boosts i couldn't image working without now.
Choosy has very good rules allowing me the choose only between the epichrome apps i want, depending on the URL.

**Comment by gregschroeder** [comment]
I highly recommend Finicky.  https://github.com/johnste/finicky

Configured via a javascript file, so a little techy, but is pretty flexible and works great.

**Comment by TwisterMc** [comment]
I've been using Browser ChooserX but I think that might also be abandoned so I started looking for something new. Right now I'm testing Browserosaurus and OpenIn. In my very limited testing, I'm liking OpenIn. I haven't ventured into a paid app yet so I can't comment on Choosy.

**Comment by dmarmor** [comment]
Thanks for this thread, guys. When I have some time, I'll try to test some of these solutions more extensively with Epichrome for compatibility, but it sounds like there are plenty of good options.

FYI, Browser Fairy has been replaced by Browser Fairy 2, which I no longer recommend, as it has been converted to a subscription model rather than outright purchase, and I find the new interface more confusing and cumbersome than the old, simple one.

I'm going to convert this issue to a discussion, as it's not directly related to an Epichrome problem.

**Comment by dmarmor** [comment]
OK, conversion seems to have worked! This will be the new home for this thread.

**Comment by TwisterMc** [comment]
To add to this, BetterTouchTool now allows you to [create custom rules for browsers](http://docs.folivora.ai/docs/1003_did_open_url.html). BTT by default is pretty customizable so it may be a contender. Thus far, I really like it even though it was a bit odd to get setup the first time.

**Comment by ylluminate** [comment]
Hey @talleux I have to say that I do like Choosy. Sorry I forgot to get back to you, but I thought better late than never. It fills some holes / deficiencies that Browser Fairy had such as arbitrary browser selection, which is much needed in this time to select a browser living in any folder vs having to be registered.
